Makefile

```diff
@@ -18,7 +18,9 @@ clean-env:
 .PHONY: install-hooks
 install-hooks: $(VENV) .pre-commit-config.yaml
 	$(VENV)/bin/pre-commit install -f --install-hooks
-	cargo fmt --help > /dev/null || rustup component add rustfmt-preview
+	cargo +nightly fmt --help > /dev/null || rustup component add rustfmt-preview
```

Comment:
According to https://github.com/rust-lang-nursery/rustfmt , we should probably install it via `rustup component add rustfmt-preview --toolchain nightly`